runtime/src/cost_model.rs

```diff
@@ -6,15 +6,16 @@
 //!
 
 use {
-    crate::block_cost_limits::*,
+    crate::{bank::Bank, block_cost_limits::*},
```

Comment:
I'm hesistant on introducing this, it's creating a cyclic dependence between our modules.

```mermaid
flowchart TD

bank;
cost_tracker;
cost_model;

bank --> cost_tracker;
cost_tracker --> cost_model;
cost_model --> bank;
```

There's not a technical reason we can't do this, but imo it points to `Bank::calculate_loaded_accounts_data_size_cost` maybe being in the "wrong" spot.

Would that function make more sense to be on `ComputeBudget` or `CostModel`?

Reply:
Good call! Was being lazy to simply want to reuse the function already in `bank`. Should move it to `CostModel`. 

Also this points out an unnecessary dependency between CostTrack and CostModel - CostTracker doesn't need to know CostModel, it only needs TransactionCost. Will clean it up in a separate PR.